rllib/BUILD

```diff
@@ -918,7 +918,7 @@ py_test(
 py_test(
     name = "tests/test_explorations",
     tags = ["tests_dir", "tests_dir_E", "explorations"],
-    size = "medium",
+    size = "large",
     srcs = ["tests/test_explorations.py"]
```

Comment:
Consider breaking up the test into smaller chunks instead of using size "large". Ideally, all tests would be sized "small". This avoids build timeouts when a large test has to be retried multiple times, and makes it easier to reproduce test failures.